components/engine_traits/src/misc.rs

```diff
@@ -187,4 +187,8 @@ pub trait MiscExt: CfNamesExt + FlowControlFactorsExt + WriteBatchExt {
 
     type DiskEngine: KvEngine;
     fn get_disk_engine(&self) -> &Self::DiskEngine;
+
+    // TODO(SpadeA): try to find a better way to reduce couple of range cache engine
+    // and kv engine
```

Comment:
=> "reduce the costs on searching the target range cache" ?

Reply:
coupling degree

Reply:
SO, it should be "reduce coupling degree" ?

Reply:
Yes, I have modified.